src/fdmdv.c

```diff
@@ -1956,6 +1956,9 @@ void fdmdv_simulate_channel(float *sig_pwr_av, COMP samples[], int nin, float ta
     float sig_pwr, target_snr_linear, noise_pwr, noise_pwr_1Hz, noise_pwr_4000Hz, noise_gain;
     int   i;
 
+    /* prevent NAN when we divide by nin below */
+    if (nin == 0) return;
+
```

Comment:
@tmiw :point_up:

Reply:
@drowe67, works for me!

Reply:
any copiled version for me to try it out?!?

Reply:
in 141 devel 800xa mode did have a differet squelch (no idea who of your clever programmers made it but it worked) ... if you now have a compiled version with new snr calculation or whatever then send me a downloadlink PLEASE ... greetz sigi dg9bfc

Reply:
@DG9BFC no.  Please be patient.  We do this as volunteers and as a service to the Ham radio community.  A new release will be available in good time.

Reply:
cant wait to try it out hi hi ;-)
(pulling your leg a bit)
i found the snr thingy and i thought its not that difficult to solve (cant you just use the snr calculation that was in 141 devel??) ... ok ok it seems i have to wait ... sniff ...
if you have something to try out ... drop me a line , send me a note, make a signal fire or whatever you can do please :-)